gatsby_fp/src/components/Layout/Body/Fields/Education.js

```diff
@@ -3,44 +3,38 @@ const shortid = require("shortid")
 
 export default function Education({ educationList }) {
 
+  const liInnerHtmlList =
+    educationList.map(element => {
+      let liString = '';
```

Comment:
Looks like we standardized on not using semicolons where optional. This is 1 of 5 similar occurrences.